CHANGELOG.md

```diff
@@ -1,15 +1,52 @@
+## 0.1.1
```

Comment:
Do you want to keep the development in `0.2.0` branch until all the packages are stabilized?

Reply:
Yes, CHANGELOG will be there once everything is finalized.